release.go

```diff
@@ -95,6 +97,16 @@ func (r Release) Version() string {
 	return r.version
 }
 
+func aggregateReleaseDeprecated(bundles []Bundle) (bool, error) {
+	for _, b := range bundles {
+		if b.Deprecated == true {
+			return true, nil
+		}
+	}
+
+	return false, nil
+}
```

Comment:
The rule is that as soon as a version bundle of a release is deprecated, the whole release is deprecated.